lua/telescope/pickers/layout_strategies.lua

```diff
@@ -33,6 +34,24 @@
 ---   flex:
 ---   - See |layout_strategies.flex|
 ---
+--- Available tweaks to the settings in layout defaults include
+--- (can be applied to horizontal and vertical layouts):
+---   mirror (default is `false`):
+---   - Flip the view of the current layout:
+---     - If using horizontal: if `true`, swaps the location of the
+---       results/prompt window and preview window
+---     - If using vertical: if `true, swaps the location of the results and
```

Comment:
Missing `

Reply:
*sigh* i am tired and it is showing

Reply:
Doesn't matter. I am super happy right now. Seeing that docgen ci work. Last thing i found after that i merge

Reply:
:heart: I appreciate it haha, I'm so glad to see the docgen working well too!